libipl/p10/common.C

```diff
@@ -413,6 +413,46 @@ static std::string ipl_get_err_msg(const ipl_error_type &err_type,
 	return ret_msg;
 }
 
+/**
+ * @brief Function to check clock redundant mode is enabled in the system
+ *
+ * if the functional clock targets is greater than one redundant mode
+ * is enabled. Default behaviour is redundant mode is disabled.
+ * Any internal failures during this function execution results  redundant
+ * mode as disabled.
+ *
+ * return true if clock redundant mode is enabled, false otherwise
+ */
+
+bool ipl_clock_redundancy_is_enabled()
```

Comment:
it would be better to change the name to ipl_clock_is_redundancy_enabled(),

Reply:
Ack